wled00/cfg.cpp

```diff
@@ -665,19 +683,23 @@ void serializeConfig() {
 #endif
 
   JsonArray nw_ins = nw.createNestedArray("ins");
+  for (size_t n = 0; n < multiWiFi.size(); n++) {
+    JsonObject wifi = nw_ins.createNestedObject();
+    wifi[F("ssid")] = multiWiFi[n].clientSSID;
```

Comment:
Can't save the SSIDs when configuring the networks, it'll always be the default one (for the first network) or empty (for the second and others).
The password seems to be saving well, so the issue seems to be either here or in `set.cpp`

Reply:
Fixed.

Reply:
> Fixed.

Still not working. I cannot connect to my network because the SSID will not save at all.

Please watch this [video](https://streamable.com/n8956l) about the issue. Also note, it doesn't care if the password is the right one or not, the SSID will still be the same ("Your_network" for the first one, and empty for the rest)

Tested with https://github.com/Aircoookie/WLED/pull/3709/commits/1bebf3d3d6f76a7afb2f7258425842f3af72bfef

Reply:
Confirm, it's working now. Well done!